plugins/catalog-backend-module-gitlab/api-report.md

```diff
@@ -33,7 +33,7 @@ export class GitlabDiscoveryEntityProvider implements EntityProvider {
 }
 
 // @public
-export class GitLabDiscoveryProcessor implements CatalogProcessor {
+export class GitlabDiscoveryProcessor implements CatalogProcessor {
```

Comment:
This is a breaking change, so users of this class will start getting failing builds.

I recommend exporting a deprecated const of the old symbols. Something like

```ts
import { GitlabDiscoveryProcessor as p } from ...;

/**
 * @public
 * @deprecated Please import as `GitlabDiscoveryProcessor` instead
 */
export const GitLabDiscoveryProcessor = p;
```

Reply:
also not sure whether it's worth changing this deprecated processor approach that got superseded by entity providers.

I would rather just leave it as it is and mark it as deprecated (in a separate PR?).